quiche/src/lib.rs

```diff
@@ -1339,6 +1347,25 @@ impl Config {
     pub fn set_disable_dcid_reuse(&mut self, v: bool) {
         self.disable_dcid_reuse = v;
     }
+
+    /// Set the server's preferred address.
+    pub fn set_preferred_address(
+        &mut self, v4: Option<SocketAddrV4>, v6: Option<SocketAddrV6>,
+        connection_id: ConnectionId<'static>, stateless_reset_token: u128,
+    ) -> Result<()> {
+        if (v4.is_none() && v6.is_none()) || connection_id.is_empty() {
+            return Err(Error::TlsFail);
```

Comment:
InvalidTransportParam is probably more intuitive for this error

Reply:
Updated 👍